policyengine_us/parameters/gov/territories/pr/tax/income/taxable_income/exemptions/dependent/amount.yaml

```diff
@@ -0,0 +1,16 @@
+description: Puerto Rico dependents exemption amount.
+JOINT:
+  2023-01-01: 2_500
+SEPARATE:
```

Comment:
Are the only values that are defined in the legal code joint and separate? If so - lets make to separate files to apply the proper ones to each filing status